pkgs/development/python-modules/free-proxy/default.nix

```diff
@@ -12,16 +13,16 @@ buildPythonPackage rec {
   version = "1.1.1";
   pyproject = true;
 
-  src = fetchFromGitHub {
-    owner = "jundymerk";
-    repo = "free-proxy";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-1hTOMbsL1089/yPZbAIs5OgjtEzCBlFv2hGi+u4hV/k=";
+  src = fetchurl {
+    # using fetchurl to avoid failed to find mirror error using fetchFromGitHub.
```

Comment:
I don't understand. This is either a general fetchFromGitHub issue which doesn't warant using fetchgit here or something specific to this package. Can you explain this in a bit more detail?

Reply:
That sounds like something wrong with their network, really. It builds on hydra. https://hydra.nixos.org/build/266126852

Reply:
so the output of the current build is cpe. that is an error from me the error is based on hash value. but when i try to change the hash value i get "error: cannot download source from any mirror"

Reply:
The short answer is, yes, your hash was wrong.

```
=> nurl https://github.com/jundymek/free-proxy/archive/refs/tags/1.1.1.tar.gz
$ nix flake prefetch --extra-experimental-features 'nix-command flakes' --json github:jundymek/free-proxy/6faf092b0bd66b5004dc2dd67f8272c311c61bbb
fetchFromGitHub {
  owner = "jundymek";
  repo = "free-proxy";
  rev = "6faf092b0bd66b5004dc2dd67f8272c311c61bbb";
  hash = "sha256-n4CdkbqLLAj+54mlVOhIuK75wAHZIlSSKNRM1OgsMS0=";
}
```

But I think your error has nothing to do with `fetchFromGitHub` itself.

Reply:
Thank you. You are right it has nothing to do with `fetchFromGitHub`.